images/oc-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -11,6 +11,24 @@ function outputToYaml() {
   set -x
 }
 
+function checkFrequency() {
+  #takes a unexpanded cron schedule, returns true if it's less often that 15 minutes
+  MINUTE=$(echo $1 | (read -a ARRAY; echo ${ARRAY[0]}) )
+  if [[ $MINUTE =~ ^(M|H|\*)\/([0-5]?[0-9])$ ]]; then
+    STEP=${BASH_REMATCH[2]}
+    if [ $STEP -ge 15 ]; then
+      echo -n true
```

Comment:
can we use proper return codes instead of an `echo`? see https://stackoverflow.com/a/19401753/2397973

Reply:
also can we rename `checkFrequency` to something that is more clear? As you expect an `true` and `false` as a return, I would call the function also something like `isScheduleMoreOftenThanEvery15Min` (just an example function name)

Reply:
👍